### DISCUSSION TITLE
[QUESTION] How to change the reserved disk space ?

### DISCUSSION THREAD

**Discussion by btdan** [longhorn/longhorn#4714] [Q&A]
## Question
Hello, I deploy Longhorn with help of helm in kubernetes cluster , just as : 
helm install longhorn \
  --namespace longhorn-system \
  --create-namespace \
  --version 1.3.1 \
.........
  --set defaultSettings.defaultDataPath="/data/longhorn/" \

In the kubernetes cluster, there is one ssd disk for each node and for longhorn to use. And It is mounted as /data
However, After longhorn is installed successfully, I find that about 1/3 disk space is reserved.

If the ssd disk is mainly used by longhorn, is it necessary to reserve so many disk space?
How to change the default value of reserved disk space?
On the above command, I just add a new parameter:
--set defaultSettings.storageMinimalAvailablePercentage=5 

And I want to reserve only 5% disk space. But after I reinstall longhorn. The parameter does not work at all. That means about 1/3 disk space is still reserved again.
Can you help me to solve this problem?
Thanks very much.

**Comment by c3y1huang** [comment]
Ref https://github.com/longhorn/longhorn/discussions/4667
